src/main/webapp/view/transcript/transcript.js

```diff
@@ -54,9 +57,14 @@ export class TranscriptLineElement extends HTMLElement {
   constructor(timestampRange, transcriptLine) {
     super();
     const contentDivElement = TranscriptLineElement.createContentDivElement();
-    TranscriptLineElement.appendParagraphToContainer(
-        timestampRange, contentDivElement,
-        ['justify-content-start', 'mb-1', 'transcript-line-timestamp']);
+    this.#timestamp =
+        TranscriptLineElement.createParagraphWithClasses(timestampRange, [
+          'justify-content-start',
+          'mb-1',
+          'transcript-line-timestamp',
+          'timestamp',
```

Comment:
what's the difference between transcript-line-timestamp and timestamp?  Can these CSS classes be merged?